scripts/rpm_owners/packages.json

```diff
@@ -0,0 +1,5586 @@
+{
+    "GConf2": {
+        "added_by": null,
+        "built_by": "centos",
+        "import_reason": null,
+        "maintainer": "OS Platform & Release",
+        "package": "GConf2",
+        "role": "extra_builddep_dep"
```

Comment:
As discussed orally, only the "maintainer" information is manageable at this level. The rest either depends on a repository state, a build, or a XCP-ng release.

For this PR, I think we should just keep package + maintainer and iterate afterwards.